README.md

```diff
@@ -25,14 +25,88 @@ and [Y.-T. Huang](https://github.com/ytdHuang).
 
 [QuantumToolbox.jl](https://github.com/qutip/QuantumToolbox.jl) is a cutting-edge Julia package designed for quantum physics simulations, closely emulating the popular Python [QuTiP](https://github.com/qutip/qutip) package. It uniquely combines the simplicity and power of Julia with advanced features like GPU acceleration and distributed computing, making simulation of quantum systems more accessible and efficient.
 
+*With this package, moving from Python to Julia for quantum physics simulations has never been easier*, due to the similar syntax and functionalities.
+
 ## Features
 
 QuantumToolbox.jl is equipped with a robust set of features:
 
-- **Quantum State and Operator Manipulation:** Easily handle quantum states and operators with a rich set of tools.
-- **Dynamical Evolution:** Advanced solvers for time evolution of quantum systems.
-- **Measurement and Statistics:** Comprehensive quantum measurement simulation and analysis.
-- **GPU and Distributed Computing:** Leverage GPU and distributed resources for high-performance computing.
+- **Quantum State and Operator Manipulation:** Easily handle quantum states and operators with a rich set of tools, with the same functionalities as QuTiP.
+- **Dynamical Evolution:** Advanced solvers for time evolution of quantum systems, thanks to the powerful [DifferentialEquations.jl](https://github.com/SciML/DifferentialEquations.jl) package.
+- **GPU Computing:** Leverage GPU resources for high-performance computing. For example, you run the master equation direclty on the GPU with the same syntax as the CPU case.
+- **Distributed Computing:** Distribute the computation over multiple nodes (e.g., a cluster). For example, you can run undreds of quantum trajectories in parallel on a cluster, with, again, the same syntax as the simple case.
+- **Easy Extension:** Easily extend the package, taking advantage of the Julia language features, like multiple dispatch and metaprogramming.
+
+## Brief example
```

Comment:
How about moving the entire `## Brief Example` after the `## Installation` ?
I think it is more nature for the users to follow: install and then how to use it.

Reply:
Let us discuss this in `zulip`?